.circleci/config.yml

```diff
@@ -33,6 +33,23 @@ jobs:
     - run:
         command: ./e2e-harness teardown
         when: always
+  e2eTestFrameworkIntegration:
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/operatorkit
+    environment:
+      MINIKUBE_VERSION: v0.25.0
```

Comment:
This can actually be removed since `e2e-harness` defaults this properly now.

Reply:
Removed.